integration/test_collection_config.py

```diff
@@ -942,14 +942,15 @@ def test_config_unknown_module(request: SubRequest) -> None:
 
 def test_create_custom_module(collection_factory: CollectionFactory) -> None:
     collection = collection_factory(
+        name="one",
```

Comment:
maybe use a global counter instead in the conftest to add to the collection names? We have the GIL, so we wont have real races

so something like
```
name_fixture = _sanitize_collection_name(request.node.name) + name + "_" +counter
counter ++
```

Reply:
Hmm, not sure I understand. The tests [were all failing](https://github.com/weaviate/weaviate-python-client/actions/runs/9775672600/job/26986616255?pr=1007) with a collection-already-exists error so there is a race happening in the test. My guess is that the GC isn't cleaning up `collection` before `collection_2` gets created now, for whatever reason, so the names need to be different

I will add the counter though, much better solution

Reply:
sorry, I moved some words around and then it didn't make sense anymore.

basically just wanted to automate that the names are unique when creating from the same test :)